en/release-notes/upgrade-guide-v2.0.md

```diff
@@ -36,32 +36,32 @@ List of known issues:
 
-## 2.1 Texture 资源配置
+## 2.1 Texture Resource Configuration
 
-也许有开发者在 Creator 1.x 注意过贴图资源的配置，比如 Wrap Mode 和 Filter Mode，但是其实在 1.x 中不论怎么设置，都是不会影响到运行时的贴图资源的。那么在 2.0 中，我们让这些配置在运行时真正生效，还增加了一个是否预乘贴图的选项：
+Maybe developers have noticed the configuration of texture resources in Creator 1.x, such as Wrap Mode and Filter Mode, but in fact, no matter how you set it in 1.x, it will not affect the runtime texture resources. So in 2.0, we made these configurations take effect at runtime, and we also added an option to prefetch textures:
 
 ![Texture Inspector](upgrade-guide-v2.0/texture.png)
 
-- Wrap Mode：循环模式，它决定了当 uv 超过 1 时如何对贴图采样
-  - Clamp：uv 的取值会自动限定在 0, 1 之间，超出直接取 0 或 1
-  - Repeat：超过时会对 uv 的值取模，使得贴图可以循环呈现
-- Filter Mode：过滤模式，它决定了对贴图像素进行浮点采样时，是否和周围像素进行混合，以达到贴图缩放时的平滑过度效果。结果上来说，Trilinear 平滑程度高于 Bilinear，高于 Point，但是 Point 很适合像素风格的游戏，在缩放贴图时，像素边界不会变得模糊，保持原始的像素画风格。
-  - Point（最近点采样）：直接使用 uv 数值上最接近的像素点
-  - Bilinear（二次线性过滤）：取 uv 对应的像素点以及周围的四个像素点的平均值
-  - Trilinear（三次线性过滤）：会在二次线性过滤的基础上，取相邻两层 mipmap 的二次线性过滤结果，进行均值计算
-- Premultiply Alpha：这是在 2.0 新增的参数，勾选时，引擎会在上传 GPU 贴图的过程中，开启 GL 预乘选项。这对于一些需要预乘的贴图非常有帮助，时常会有一些用户对于贴图周围或者文字周围莫名其妙的白边无法理解，这是贴图周围的半透明像素造成的：
+- __Wrap Mode:__ Loop mode, which determines how the texture is sampled when uv exceeds 1.
+  - __Clamp:__ the value of __uv__ is automatically limited to __0, 1__ and exceeds __0 or 1__ directly.
+  - __Repeat:__ When over, the value of __uv__ is modulo, so that the texture can be rendered in a loop.
+- __Filter Mode:__ Filter mode, which determines whether to blend the surrounding pixels with the surrounding pixels when floating point samples are used to achieve the smoothing effect of texture scaling. In effect, Trilinear smoothness is higher than Bilinear, higher than Point, but Point is very suitable for pixel-style games. When scaling textures, the pixel boundaries will not be blurred, maintaining the original pixel painting style.
+  - __Point (nearest point sampling):__ directly use the nearest pixel on the uv value
+  - __Bilinear (secondary linear filtering):__ take the average of the pixel corresponding to uv and the surrounding four pixels
+  - __Trilinear (triangular linear filtering):__ Based on the quadratic linear filtering, the quadratic linear filtering results of two adjacent mipmaps are taken for the mean calculation.
+- __Premultiply Alpha:__ This is a new parameter in 2.0. When checked, the engine will enable the GL pre-multiply option during the upload of the GPU map. This is very helpful for some textures that need to be pre-multiplied. Often there are some users who can't understand the inexplicable white edges around the texture or around the text, which is caused by the semi-transparent pixels around the texture:
 
-![Spine 骨骼接缝处的奇怪白边](upgrade-guide-v2.0/spine-border.png)
+![Spine's strange white edges at the bone seams] (upgrade-guide-v2.0/spine-border.png)
```

Comment:
![Spine's strange white edges at the bone seams](upgrade-guide-v2.0/spine-border.png)

Reply:
Thanks. I thought I found all of these when reviewed the formatting. fixed.